Header header 

sensor_msgs/Image rawImg        # 原始的图像信息

FeaturePoints featurePoints     # 包含的点特征信息

FeatureLines featureLines       # 包含的线特征信息
